staging/src/k8s.io/code-generator/cmd/validation-gen/testscheme/testscheme.go

```diff
@@ -243,9 +243,10 @@ func (s *ValidationTestBuilder) Value(value any) *ValidationTester {
 // tests for a validatable value.
 type ValidationTester struct {
 	*ValidationTestBuilder
-	value    any
-	oldValue any
-	opts     sets.Set[string]
+	value       any
+	oldValue    any
+	opts        sets.Set[string]
+	subresource string
```

Comment:
Q: is it ok if this is a string when Request.Subresources is a []string?  Does this support testing nested subresources?

Reply:
I'll make this a slice for consistency and extension apiserver dev.  We don't have any in-tree sub-sub-resources.